docs/2.7.6/docs/app-dev/explicit-contract-disclosure.rst

```diff
@@ -3,7 +3,7 @@
 
 .. _explicit-contract-disclosure:
 
-Explicit Contract Disclosure (Alpha)
+Explicit Contract Disclosure (Beta)
```

Comment:
@hrischuk-da Can we do this? Or does this technically trigger 2.7.7? Tudor asked me about this but I don't know.

(@tudor-da I haven't done a full review yet; just asking this Q for now)

Reply:
Oh, I guess I was looking at an outdated file; you can disregard my comment